README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# MACS 20500 HW01
+
+# Brief Biography
+
+## Name
+**Yuhan Jin**
+
+## Some facts about me 
+
+* I'm from China
+* a first-year **MAPSS** student with a concentration on **Economics**. 
+* I obtained my B.S.degree in Economics at Uninversity of Minnesota, Twin Cities
+* I am interested in financial economy and international trade
+
+## Links
+* My facebook page is:https://www.facebook.com/yuhan.jin.18
```

Comment:
The link here works, but if you want to embed the link in text you could write the code as:
`This is my [Facebook page](https://www.facebook.com/yuhan.jin.18)`